content/slides/symbology-with-qgis.md

```diff
@@ -7,25 +7,183 @@ title-slide-attributes:
   data-background-image: "/_media/DMS_1842643_12758_20180418_18111267_clipped.jpg"
 ---
 
-## TODO
-<!-- TODO:
-* Symbolizing geospatial data in QGIS
-    * Colormaps (15 minutes)
-        * `cmocean` as a strong default
-            * How to add `cmocean` and other cmaps to QGIS (_TODO: add link to video_)
-        * Common mistakes in choosing a colormap
-            * Basemap compatibility
-                * Colorbrewer (a useful tool, but very specific use-case)
-            * Jet / rainbow
-        * Colormap analysis with `viscm`. _TODO: consider utilizing
-          <https://github.com/nsidc/colormap-analysis>_
-    * Representing vector data attributes (10 minutes)
-        * Point markers
-        * Color
-        * Size
-        * Labels
-        * Scale dependent rendering
-        * Grouping
-    * Accessibility related concerns?
-    * Exporting layer settings as QLR
--->
+<!-- alex disable color colors -->
+
+## Symbology is important
+
+Symbology is [critical](https://www.kennethmoreland.com/color-advice/BadColorMaps.pdf)
+to understanding data! Consider:
+
+* Does your colormap look uniform across your values?
+* Do your symbology choices unintentionally exclude any users, for example people with
+  Color Vision Deficiency (CVD), a.k.a. colorblindness?
+* What, aside from your data, will be in your visualization?
+    * Basemap?
+    * Other data elements?
+
+::: {.notes}
+* "Perceptual uniformity" is important for even users with perfect sight. The "Jet"
+  colormap suffers
+* Look for colormaps that are friendly to all forms of color deficiency
+* Ensure contrast with your basemap: A light basemap requires a colormap with saturated
+  (colorful) light values, and a dark basemap requires a colormap with saturated dark
+  values.
+* Ensure contrast with your other data elements. Red dots representing ice core
+  locations won't show up well over temperature data displayed with the `heat` colormap.
+:::
+
+
+# Colormaps
```

Comment:
Later on we have a section on vector symbology. Should we do that here? Two main topics being colormaps and maybe contour representation of raster data?

I suppose colormap discussion could also apply to vector data if e.g., points are colored according to an attribute.

Reply:
Decided to add speaker notes on the colormap slide about how this topic can apply to both rasters and vectors, but is usually the main concern when visualizing rasters.